tests/test_bootstrap.py

```diff
@@ -1027,6 +1046,7 @@ def test_log_error_invokeId_line_framed_log_sink(self):
                     "testrequestId",
                 )
                 bootstrap.log_error(err_to_log, log_sink)
+            after = int(time.time_ns() / 1000)
```

Comment:
nit: adding randomness into unit tests is discouraged. In this particular case it should not cause losing determinism, so it's fine to leave if here